dolphinscheduler-task-plugin/dolphinscheduler-task-datax/src/main/java/org/apache/dolphinscheduler/plugin/task/datax/DataxTask.java

```diff
@@ -167,6 +167,7 @@ public void handle() throws Exception {
             setAppIds(commandExecuteResult.getAppIds());
             setProcessId(commandExecuteResult.getProcessId());
         } catch (Exception e) {
+            logger.error("datax task error", e);
```

Comment:
In fact, the exception will be thrown to upper layer, and will log at the upper layer. I don't suggest we log the exception and throw it again, this will cause the exception occur multiple times in log.